File: PosiljkaApp_video6c/app/src/main/java/android/fit/ba/posiljka/data/KorisnikPregledVM.java
package android.fit.ba.posiljka.data;

import java.io.Serializable;
import java.util.List;

public class KorisnikPregledVM implements Serializable
    {
        public static class Row implements Serializable
        {
            public Integer id;
            public String ime;
            public String prezime;
            public String opstina;
            public String drzava;
        }
        public List<Row> rows;
    }

File: PosiljkaWebApi_video5_video6/Posiljka.Web/ViewModels/api/KorisnikPregledVM.cs
using System.Collections.Generic;

namespace Posiljka.Web.ViewModels.api
{
    public class KorisnikPregledVM
    {
        public class Row
        {
            public int? id;
            public string ime;
            public string prezime;
            public string opstina;
        }

        public List<Row> rows;
    }
}
